Fruitties/androidApp/src/main/java/com/example/fruitties/android/MyApplicationTheme.kt

```diff
@@ -61,10 +66,23 @@ fun MyApplicationTheme(
         large = RoundedCornerShape(0.dp),
     )
 
-    MaterialTheme(
-        colorScheme = colors,
-        typography = typography,
-        shapes = shapes,
-        content = content,
+    val topAppBarColors = TopAppBarColors(
+        containerColor = colors.primary,
+        scrolledContainerColor = colors.primary,
+        navigationIconContentColor = colors.onPrimary,
+        titleContentColor = colors.onPrimary,
+        actionIconContentColor = colors.onPrimary,
     )
+
+    CompositionLocalProvider(LocalTopAppBarColors provides topAppBarColors) {
+        MaterialTheme(
+            colorScheme = colors,
+            typography = typography,
+            shapes = shapes,
+            content = content,
+        )
+    }
 }
+
+@OptIn(ExperimentalMaterial3Api::class)
+val LocalTopAppBarColors = compositionLocalOf<TopAppBarColors?> { null }
```

Comment:
It seems a bit suspicious to create a `CompositionLocal` here just for a single usage, can these custom colors just be specified inline?

Reply:
I agree, I now moved the colors into a simple argument since it's only used in the one place.